Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -406,8 +406,19 @@ body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
         --slider-track-background: hsl(0, 0%, 35%);
         --slider-track-box-shadow: inset 0 0 3px hsl(0, 0%, 20%);
         --slider-track-tick-background: hsl(0, 0%, 55%);
+
+        --high-contrast-background-color: hsl(0, 0%, 10%);
+        --high-contrast-border-color: hsl(0, 0%, 10%);
+        --high-contrast-text-color: hsl(0, 0%, 90%);
```

Comment:
if these are only used once i think we can eliminate them and inline them instead

also, im not sure we've ever considered nesting `@media` like this before.  i feel like we'd prefer it to be at the top-level with an `and`, but im not entirely against this style

```css
@media (prefers-color-scheme: dark) and (prefers-contrast: more) {
    :root {
        --background-color: hsl(0, 0%, 10%);
        --border-color: hsl(0, 0%, 10%);
        --text-color: hsl(0, 0%, 90%);
    }
}
```

Reply:
The better way to use nesting here is:
```
@media (prefers-color-scheme: dark) and (prefers-contrast: more) {
  --background-color: hsl(0, 0%, 10%);
  --border-color: hsl(0, 0%, 10%);
  --text-color: hsl(0, 0%, 90%);
 }
 ```
 `:root` is implicit because of the parent rule!